packages/ai/src/utils/streaming-text-response.ts

```diff
@@ -0,0 +1,37 @@
+import { StreamingTextResponse as _StreamingTextResponse } from "ai";
+
+export type StreamingTextResponseType = StreamingTextResponse;
+
+// vercel/ai's StreamingTextResponse does not include request.headers.raw()
+// which @vercel/remix uses when deployed on vercel.
+// Therefore we use a custom one.
+export class StreamingTextResponse extends _StreamingTextResponse {
```

Comment:
same thing, using underscore doesn't explain why we have classes with the same name, it just raises questions, I would rather have them more explicit names that explain why they exist

Reply:
yes, but reflecting it in the name is important still, e.g.
```ts
export class StreamingTextResponse extends RemixStreamingTextResponse
```

Reply:
even better now its even clear what wasn't before, that you are creating a remix interface that extends the vercel interface ... I completely didn't know that before, even with comment